ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -129,3 +129,5 @@
 {% include 'documentation/eos-cli.j2' %}
 {# Custom Templates #}
 {% include 'documentation/custom-templates.j2' %}
+{# COPP Policy Maps #}
```

Comment:
This should be arranged together with the other policy maps and not at the end of the file. Preferable put it into the same template as policy maps and just add another section. Your headings in the doc template starts at level-3 (###) which does not fit this place.

Reply:
Fixed.